Mathlib/Data/ENat/Basic.lean

```diff
@@ -405,60 +405,50 @@ protected theorem map_add {β F} [Add β] [FunLike F ℕ β] [AddHomClass F ℕ
     (f : F) (a b : ℕ∞) : (a + b).map f = a.map f + b.map f :=
   WithTop.map_add f a b
 
+@[to_additive]
+theorem map_eq_one_iff {β} {f : ℕ → β} {v : ℕ∞} [One β] :
+    ENat.map f v = 1 ↔ ∃ x, v = .some x ∧ f x = 1 := WithTop.map_eq_one_iff
+
+@[to_additive]
+theorem one_eq_map_iff {β} {f : ℕ → β} {v : ℕ∞} [One β] :
+    1 = ENat.map f v ↔ ∃ x, v = .some x ∧ f x = 1 := WithTop.one_eq_map_iff
+
 /-- A version of `ENat.map` for `OneHom`s. -/
--- @[to_additive (attr := simps (config := .asFn))
---   "A version of `ENat.map` for `ZeroHom`s"]
+@[simps (config := .asFn)]
 protected def _root_.OneHom.ENatMap {N : Type*} [One N] (f : OneHom ℕ N) :
```

Comment:
Why do we want to `protect` this `def`? The name seems specific enough, no? (same question for the `def` below).

Reply:
This was just copied from `WithTop`, I can remove this